runtime/tests/decimal.c

```diff
@@ -0,0 +1,49 @@
+#include <string.h>
+#include <assert.h>
+#include "../balrt.h"
+#include "../hash.h"
+#include "../third-party/decNumber/decQuad.h"
+
+HASH_DEFINE_KEY;
+
+void validate(TaggedPtr tp, const char *decStr) {
```

Comment:
Call this file something other than decimal.c to avoid confusion with file in parent directory. Also add filename to .gitignore.

Reply:
Fixed